Modal styling

KennethEllebring opened the pull request:
- added teanary string for choosing "like" vs "likes" if its more than 1 like, also removed header from post modal
- added a few classnames and changed some in edit modal
- added stylcheet path to feed.js
- added scss code for post/edit modals

arapg commented on src/styles/Popup_modal.scss:
```diff
@@ -0,0 +1,191 @@
+      bottom: 18%;
+      display: flex;
+      margin-bottom: 1rem;
+      .new-comment-input {
```
Maybe add in `border: none;`?

KennethEllebring replied:
removed borders on both edit posts and post comments

arapg commented on src/styles/Popup_modal.scss:
```diff
@@ -0,0 +1,191 @@
+      color: #ffffff;
+      margin-bottom: 1rem;
+    }
+    .edit-comment {
```
Shouldn't it be `.edit-post`? ;-)

KennethEllebring replied:
changed names to match edit post instead of comment

KennethEllebring commented:
- changed the ugly windows scroll with a newer modern look
- changed the scroll to only scroll comments
- changed some classnames that were confusing
- moved the input comment inside comment post window

AlexSaxena reviewed (approved):
Snyggt jobbat! Gillar nya scrollbaren 👌
